components/RuiExpandCollapse/src/RuiExpandCollapse.css.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright Deloitte Digital 2019
+ *
+ * This source code is licensed under the BSD-3-Clause license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {css} from 'lit-element';
+
+/**
+ *  RuiExpandCollapse Variables
+ */
+export const variables = css`
+	:host {
+		/**
+		 * @variable The expanded section background colour
+		 */
+		--bg: #F2F2F2;
+
+		/**
+		 * @variable The exapnd section text colour
+		 */
+		--txt: #000000;
```

Comment:
We are using `--txt` here but RUIButton uses `--color`.

Same story for background. I kinda prefer the variable name that is a bit closer to the underlying property name it's getting applied to.

I really want to be consistent in naming our variables, can you please document a css variable naming style guide in the CONTRIBUTING.md file for common things like color, background, padding and margin. Then make sure we are consistent throughout.

Reply:
@jennasalau Okay I have made the variable naming more consistent throughout and added some docs for it, let me know your thoughts. Basically its <scope?>-<property>-<modifier?>. 
Most of the code was already using a variation of this, just needed to make the ordering of words more consistent.